src/components/tree-views-container.tsx

```diff
@@ -33,16 +42,16 @@ const initialMousePosition = {
 // Node.js (used for tests) version < 11 has no Object.fromEntries
 Object.fromEntries =
     Object.fromEntries ||
-    ((arr) =>
-        arr.reduce((acc, [k, v]) => {
+    ((arr: any) =>
+        arr.reduce((acc: any, [k, v]: any) => {
             acc[k] = v;
             return acc;
         }, {}));
 
-function buildPathToFromMap(nodeId, mapDataRef) {
+function buildPathToFromMap(nodeId: UUID | undefined, mapDataRef: Record<string, IDirectory> | undefined) {
     let path = [];
     if (mapDataRef && nodeId) {
-        let currentUuid = nodeId;
+        let currentUuid: UUID | null = nodeId;
```

Comment:
`nodeId` can't be `null` but `undefined`
```suggestion
        let currentUuid: UUID | null = nodeId ?? null;
```

Reply:
Done